src/ccover/ccover_bmc.h

```diff
@@ -0,0 +1,107 @@
+/*******************************************************************\
+
+Module: Bounded Model Checking for Test Suite Generation
+
+Author: Daniel Kroening, kroening@kroening.com
+
+\*******************************************************************/
+
+/// \file
+/// Bounded Model Checking for Test Suite Generation
+
+#ifndef CPROVER_CCOVER_CCOVER_BMC_H
+#define CPROVER_CCOVER_CCOVER_BMC_H
+
+#include <list>
+#include <map>
```

Comment:
I don't think either of those are used here, but `<memory>` is needed.

Reply:
done